erroranalysis/requirements.txt

```diff
@@ -2,4 +2,4 @@ numpy>=1.17.2
 pandas>=0.25.1
 scipy>=1.4.1
 scikit-learn>=0.22.1
-lightgbm>=3.1.1
\ No newline at end of file
+lightgbm>=2.0.11
```

Comment:
Why the magic version 2.0.11? Seems like there are even versions here: https://pypi.org/project/lightgbm/#history?

Reply:
That's the one required by automl

Reply:
or rather: the minimum required by automl

Reply:
oh. I thought it was 2.3.0. That is the upper bound.